hudi-utilities/src/main/java/org/apache/hudi/utilities/streamer/HoodieStreamer.java

```diff
@@ -261,9 +270,19 @@ public static class Config implements Serializable {
         + " to break ties between records with same key in input data. Default: 'ts' holding unix timestamp of record")
     public String sourceOrderingField = "ts";
 
-    @Parameter(names = {"--payload-class"}, description = "subclass of HoodieRecordPayload, that works off "
-        + "a GenericRecord. Implement your own, if you want to do something other than overwriting existing value")
-    public String payloadClassName = OverwriteWithLatestAvroPayload.class.getName();
+    @Parameter(names = {"--payload-class"}, description = "Deprecated. Use --merge-mode for overwite or event time merging."
+        + " Subclass of HoodieRecordPayload, that works off a GenericRecord. Implement your own, if you want to do something "
+        + "other than overwriting existing value")
+    public String payloadClassName = null;
+
+    @Parameter(names = {"--merge-mode", "--record-merge-mode"}, description = "mode to merge records with")
+    public RecordMergeMode recordMergeMode = null;
+    
+    @Parameter(names = {"--merger-strategy", "--record-merger-strategy"}, description = "only set this if you are using custom merge mode")
```

Comment:
For new configs or cmd arguments added, let's make the naming consistent.
```suggestion
    @Parameter(names = {"--merger-strategy-id", "--record-merge-strategy-id"}, description = "only set this if you are using custom merge mode")
```